tests/Authorization.AspNetCore.Tests/Authorization.AspNetCore.Tests.csproj

```diff
@@ -2,15 +2,15 @@
   <Import Project="../../Tests.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net6;net5;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net7;net6;net5;netcoreapp3.1</TargetFrameworks>
     <Nullable>disable</Nullable>
     <NoWarn>$(NoWarn);CS0618</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <ProjectReference Include="..\..\src\Authorization.AspNetCore\Authorization.AspNetCore.csproj" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.*" />
```

Comment:
Maybe switch to `x.y.*` versions for all MS packages in test projects and samples here and in other repos to minimize noise from dependabot?

Reply:
Yes that was the goal here